### DISCUSSION TITLE
My Github page is not deploying

### DISCUSSION THREAD

**Discussion by Evan260** [community/community#119992] [Other Feature Feedback, Questions, & Ideas]
As you can see in the images below, I have a simple website in my root folder in a repo, and I went to the pages tab in the settings menu to publish the website, however, I am not getting the option to publish it. I am fairly new to Github, so perhaps I am missing something.

![Screenshot 2024-04-17 115502](https://github.com/community/community/assets/109839359/e0d77da3-966a-4611-8062-ec39a61b92c3)
![Screenshot 2024-04-17 115508](https://github.com/community/community/assets/109839359/eaf6b2ac-2c6e-472b-b544-80b2ce8e3ccc)

**Comment by LiteBrite82** [comment]
Hi @Nebula580... 

You should just have to pick the correct branch, select root, then push save. By default GH Pages will select the file named index.html.

**Reply by LiteBrite82** [reply]
I was able to clone your repo and get it to deploy with the above process. It took 10 minutes for it to finally show up, but it did deploy.

**Reply by LiteBrite82** [reply]
refresh the browser?

**Reply by LiteBrite82** [reply]
https://github.com/LiteBrite82/testing

**Reply by LiteBrite82** [reply]
I will delete after testing

**Comment by Evan260** [comment]
Success! Thanks for the help. I went into the actions menu and clicked re-run all jobs, which fixed it.

![Screenshot 2024-04-17 192732](https://github.com/community/community/assets/109839359/7136b542-ce50-4a02-8feb-c2c60beac08a)
![Screenshot 2024-04-17 192727](https://github.com/community/community/assets/109839359/d9f84ea5-d2df-4ebd-a427-b9f7c7571e65)
